### ISSUE TITLE
Need integration test

### ISSUE THREAD

**Issue by guoger** [guoger/stupid#15]
There's zero integration today in this project. User/Dev must manually spin up a fabric network and try it out. We need a proper integration test that spins up **real** fabric network and run stupid with light traffic against it

**Comment by SamYuan1990** [comment]
deal to https://github.com/MicrosoftDocs/azure-devops-docs/issues/8178
seems that we can't having peer container adding into `/etc/hosts` for pipeline?

**Comment by guoger** [comment]
fabric integration test starts peer/orderer as local process, which we could easily "borrow".

it's probably not so easy to run containers in azp, as the test itself is already contained in containers...

**Comment by SamYuan1990** [comment]
well, I am ensure, azp able to run docker , as 
```
https://github.com/SamYuan1990/FabricJavaPool/blob/azure-pipelines/cicd.sh
```

You want to reference fabric/fabric-go-sdk, they basing on golang, so a sub class... 
This project need maintain a specific go file for testing...

if use fabric-sample, it suppose to be shell script as user manual.

Either way is fine to meat the goal.

We need a proper integration test that spins up real fabric network and run stupid with light traffic against it...

**Comment by guoger** [comment]
fixed in #21
